solutions/core-landing-zone/mgmt-project/org-policies/compute-disable-serial-port-logging-except-mgt-project.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+#
+# GCP Organization Policies
+# Org policies that correspond with a Guardrail will contain a label indicating what Guardrails it helps in enforcing
+# https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints
+#
+# Constraint: compute.disableSerialPortLogging
+#
+# This boolean constraint disables serial port logging to Stackdriver from Compute Engine VMs belonging to the
+# organization, project, or folder where this constraint is being enforced.
+#
+# By default, serial port logging for Compute Engine VMs is disabled, and can be selectively enabled on a per-VM or
+# per-project basis using metadata attributes. When enforced, this constraint disables serial port logging for new
+# Compute Engine VMs whenever a new VM is created, as well as preventing users from changing the metadata attribute
+# of any VMs (old or new) to True. Disabling serial port logging can cause certain services that rely on it, such
+# as GKE Autopilot, to not function correctly.
+#
+# The exception on Management project is needed for GKE Autopilot autoscaling to function properly.
+#
+#########
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: ResourceManagerPolicy
+metadata:
+  name: compute-disable-serial-port-logging-except-mgt-project
```

Comment:
Maybe change add a `kpt-set` annotation to this to prevent collisions with other resources.
`kpt-set: compute-disable-serial-port-logging-except-${management-project-id}`

Reply:
Done.